workshops/_template/facilitation-guide.md

```diff
@@ -0,0 +1,28 @@
+# Facilitator's Guide
+## <Workshop Name>
+
+<workshop description>
+
+Duration: XX hours
+Date Updated: XXXX-XX-XX
+
+### Goal
+
+## Materials
+
+Materials needed for the workshop include:
+* 
+*
+*
+
+The repository is structured as follows:
+* `<folder-name>/`
+* `<folder-name>/`
+* `<folder-name>/` with [direct link](#) to important resources
```

Comment:
I just wanted to have an example of a link just because sometimes I'm lazy! The hash placeholded would just make it open to the same page... which I guess doesn't make much sense to anyone but me.